### ISSUE TITLE
Add more color options

### ISSUE THREAD

**Issue by willuhmjs** [caleb531/faith-dashboard#2]
### Feature Request
There are only six colors and more could be added; I can dev this and create a PR.

**Comment by caleb531** [comment]
@willuhm-js What other colors were you thinking of adding? I was a little wary of adding colors like red and orange because they might be too vibrant and distracting. Similarly, neutral colors like gray/black would be too dull and depressing. What are your thoughts?

**Comment by willuhmjs** [comment]
You can add red and orange, just use the same saturation and lightness while changing the hue. In laymans terms, use a comfy shade of red and orange instead of a vibrant shade.

**Comment by caleb531** [comment]
@willuhm-js Fair point. Well, if you want to submit me a PR for red & orange (along with any other colors you think would be worth having), please go ahead. I think I actually already have a `$red` color variable defined in `_colors.scss`, so maybe start there to see how that works as an overall theme color.

Files you'll need to modify:

`components/app/ThemeSwitcher.tsx` (`themeList` array)
`components/app/app.d.ts` (see `AppTheme` string union)
`styles/base/_colors.scss` (define any needed color variables and add them to `$theme-colors` map)

**Comment by willuhmjs** [comment]
I was able to add orange. I tried to add two other colors and it displayed the colors as gray. Don't know what's up with that.

**Comment by caleb531** [comment]
@willuhm-js Grey is the fallback background color, so if you ever see it, it probably means you didn't define the theme colors property. The code you wrote your "orange color" PR looks correct, so maybe use that as a reference.

**Comment by willuhmjs** [comment]
Issue resolved.
